### ISSUE TITLE
ImportError in Python version: ImportError: /usr/local/lib/python3.6/site-packages/asc.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN5anyks3Idw4NIDWE

### ISSUE THREAD

**Issue by 32r81b** [anyks/asc#5]
You may repeat after me using docker image:

docker run -it python:3.6 bash
python3 -m pip install pybind11
python3 -m pip install anyks-sc

All steps finished withot errors.

Check installed libs:
![image](https://user-images.githubusercontent.com/8361949/145672231-d530efff-5567-4746-af5a-8877a16440cc.png)

Try import asc: 
![image](https://user-images.githubusercontent.com/8361949/145672168-9cd5697f-d2f9-49fe-bf50-523094a64331.png)

**Comment by anyks** [comment]
Good day! This is not the same library, similar in name and the python interpreter confuses them.

Perhaps this is an old version of the library.
Try uninstalling the old version and building again.

**Comment by 32r81b** [comment]
Thank you, for your respond.
What actually I have to do?

pip uninstall anyks-sc
pip install anyks-sc


Or I need to build anyks from git repository?

git clone --recursive https://github.com/anyks/asc.git
mkdir ./build
cd ./build
cmake ..
make

**Comment by anyks** [comment]
If you need a python version, the build is done via pip only. Building from the repository only for the standalone application.

**Comment by anyks** [comment]
>> Thank you, for your respond.
>> What actually I have to do?
>> 
>> pip uninstall anyks-sc
>> pip install anyks-sc

Yes, everything is relevant, under Linux and FreeBSD everything should be assembled correctly, I checked it on all current distributions. If the package compiled, then it should work.
